### DISCUSSION TITLE
2023/11/13/%E9%94%90%E8%AF%84/WHITE%20ALBUM%202/

### DISCUSSION THREAD

**Discussion by giscus** [dangokanako/blog-comments-gitalk#23] [Announcements]
# 2023/11/13/%E9%94%90%E8%AF%84/WHITE%20ALBUM%202/

“那一年冬天，我在睡梦中时泪流不止。醒来后家母问我咋了，我摇摇头说不记得，我不好意思告诉她，我梦见了一个游戏里的场景，我代入进去了。”                                       ————谭亭  前言虽然我有写长评的习惯，但白2我确实不配写。 这个长达甚至一百小时的文本小说，每个词语都蕴含着深沉的爱意，每行句子都承载着思念的重量，随处稍微挖掘一下，思绪与感情便如涌

https://dangokanako.github.io/2023/11/13/%E9%94%90%E8%AF%84/WHITE%20ALBUM%202/

**Comment by gpdxlss630** [comment]
小鸟酱多多施工喵，😘
